playbooks/roles/heketi-storage/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+- name: retrieve gluster-pod IPs
```

Comment:
we have too many different directories hekety, hekety-storage, gluster-server-heketi, please make a single one